### ISSUE TITLE
overflow={true} still can`t be shown

### ISSUE THREAD

**Issue by SkyLin0909** [jasonslyvia/react-lazyload#172]
lazy loading components inside a overflow container, set this to true.
```
 <LazyLoad height={50} offset={100} overflow once>
       <img src={item.headImg} alt="" className={style.avatar} />
 </LazyLoad>
```
but, not render

**Comment by jasonslyvia** [comment]
try remove `offset` props and see what's happening

**Comment by SkyLin0909** [comment]
Still can't show。
```
{listObj[e].map(v => (
                  <li
                    key={v.targetId}
                    className={
                      v.targetId === contactListActiveId ? style.active : ''
                    }
                    onClick={() => this.checkDetailInfo(v)}
                  >
                    <LazyLoad height={50} overflow once>
                      <img src={v.headImg} alt="头像" />
                    </LazyLoad>
                    <p className={style.renderName}>{v.renderName}</p>
                  </li>
                ))}
```

**Comment by jasonslyvia** [comment]
which version of lazy load are you using?

**Comment by SkyLin0909** [comment]
"react": "^16.2.0",
 "react-lazyload": "^2.3.0",

**Comment by ranneyd** [comment]
+1 same for me

**Comment by jasonslyvia** [comment]
any chance you can provide a working demo? ideally a [fiddle](https://jsfiddle.net) would be very much helpful to debug this issue.

**Comment by ninio** [comment]
I have a similar issue:
When an element is both in an overflow element and this element is outside of viewport ->
Elements are loaded only after a scroll of the overflow container.

They are not visualized when the overflow container comes into view.

It is probably due to the fact that the scroll event is attached in the else statement of `if( this.props.overflow )` - [https://github.com/jasonslyvia/react-lazyload/blob/552315242635d0bf36c5387d644e6eded6641ce6/src/index.jsx#L219](index.js:219)

**Comment by SkyLin0909** [comment]
@ninio Did you solve this bug with this method?

**Comment by ninio** [comment]
It solved it for me, yes.

Mind that the parent of LazyLoad should be the only overflowing container, apart from the window. So you should have scrollbars only on the window and the parent container. 
If you have more nested overflow containers (with scrollbars) this would still not work in all the cases.

**Comment by SkyLin0909** [comment]
@ninio Yes, I only have one scrollable container，but now， whether to wait for the author's consent your`s PR?

**Comment by SkyLin0909** [comment]
@ninio The author has not yet approved your merger，I am going to use your changes and fix them directly in the source code

**Comment by ninio** [comment]
You can do that and update later (once the PR is approved)

**Comment by mikevaleriano** [comment]
@ninio I've installed your version but for the life of me I still can't get it to work inside a div with overflow. There are no other components in the tree with overflow set.

I'm using styled-components and my overflow container is this:
```js
const TabContent = styled.div`
max-height: 65vh;
overflow-x: hidden;
overflow-y: auto;
`;
```
The component being lazy loaded is:
```js
const Thumbnail = props => (
  <React.Fragment>
    <LazyLoad height="120px" overflow once>
      <img
        className="materialboxed"
        src={props.img}
        alt={props.alt}
        title={props.alt}
        style={{ maxWidth: "100px" }}
        onLoad={e => Materialize.Materialbox.init(e.target)}
      />
    </LazyLoad>
    <figcaption>{props.alt}</figcaption>
  </React.Fragment>
);
```

The content lazy loads just fine without the overflow in my container. What's the correct way to achieve that with overflow? The docs say my overflow container needs to have a `position` property set to a value other than `static`; neither `relative` nor `absolute` work.
